src/lib/tls/asio/asio_stream.h

```diff
@@ -627,25 +627,29 @@ class Stream
                const std::string& hostname,
                const TLS::Policy& policy) override
                {
-               if(m_stream.m_context.has_verify_callback())
+               if(m_context.has_verify_callback())
                   {
-                  m_stream.m_context.get_verify_callback()(cert_chain, ocsp_responses, trusted_roots, usage, hostname, policy);
+                  m_context.get_verify_callback()(cert_chain, ocsp_responses, trusted_roots, usage, hostname, policy);
                   }
                else
                   {
                   Callbacks::tls_verify_cert_chain(cert_chain, ocsp_responses, trusted_roots, usage, hostname, policy);
                   }
                }
 
+            bool shutdown_received;
+            boost::beast::flat_buffer receive_buffer;
+            boost::beast::flat_buffer send_buffer;
```

Comment:
should those be prefixed with `m_`?

Reply:
I removed the prefix because they are public in the StreamCore. StreamCore is a nested class that is only accessible to Stream though.